README.md

```diff
@@ -25,11 +25,28 @@ Multiresource yaml files to deploy to plain kubernetes.
 ### Using the Operator
  - Clone `kaoto-operator` repository 
  - Run `make deploy` which creates `kaoto-operator` project and deploy all necessary resources as Kaoto CRD and necessary `serviceaccounts`
- - Deploy Kaoto Custom Resource sample: `oc apply -f config/samples/_v1alpha1_kaoto.yaml`
+ - Deploy Kaoto Custom Resource sample: `kubectl apply -f config/samples/designer_v1alpha1_kaoto.yaml`
 
  
 ### Via Operator Hub 
   - Install Kaoto operator catalog resource:  
     ```oc apply -f https://raw.githubusercontent.com/KaotoIO/kaoto-operator/main/catalogSource.yaml```
  - Install the Kaoto Operator from the Operator
  - Create Kaoto instance from the Kaoto Operator page
+
+## Local development
+
+### Run Operator inside the cluster
+1. Start minikube: `minikube start`
```

Comment:
can you also include instructions to set-up the ingress addon?

Reply:
doen